enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -196,6 +198,11 @@ func (mc *MigrationCache) Contains(ctx context.Context, d *repb.Digest) (bool, e
 		log.Warningf("Migration digest %v src contains %v, dest contains %v", d, srcContains, dstContains)
 	}
 
+	if doubleRead && srcContains && !dstContains {
+		metrics.MigrationNotFoundErrorCount.With(prometheus.Labels{metrics.NotFoundErrorType: "contains"}).Inc()
+		metrics.MigrationNotFoundErrorCount.With(prometheus.Labels{metrics.NotFoundErrorType: "all"}).Inc()
```

Comment:
There's no need for an `all` value since the data can be aggregated in prometheus / grafana.

Reply:
Done